test/terraform/aws/main.tf

```
@@ -0,0 +1,102 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "materialize_infrastructure" {
+  source = "git::https://github.com/MaterializeInc/terraform-aws-materialize.git"
```

Probably best to pin this to a specific tag in case that we make some breaking changes in the future:

```suggestion
  source = "git::https://github.com/MaterializeInc/terraform-aws-materialize.git?ref=v0.1.3"
```